EarlyBird/CoursesTableViewController.swift

```diff
@@ -24,4 +28,57 @@ class CoursesTableViewController: UITableViewController {
         cell!.textLabel!.text = courseNames[indexPath.row]
         return cell!
     }
+    
+    func makeGetRequest() {
+        // Set up the URL request for one specific course to before tring all courses
+        let courseEndpoint: String = "https://early-bird-courses.herokuapp.com/api/v1/courses/1"
```

Comment:
@matthewecampbell I know that you don't always have to specify data type, but in this instance it is required.  I have been letting the compiler guide me, but I will look into this and get back to you.